### PR TITLE
Add JSDoc annotations to templates

### PR DESCRIPTION
**Pull request by Rich-Harris**
This PR builds on #4495, adding JSDoc annotations to template source files alongside the TypeScript annotations. These are stripped out of the TypeScript build and the no-typechecking build, while the TypeScript is stripped out of the JSDoc build and the no-typechecking build.

### CODE REVIEW

File: packages/create-svelte/index.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ function write_template_files(template, typescript, name, cwd) {
 }
 
 /**
- *
+ build*
```
Comment (benmccann) [inline packages/create-svelte/index.js:43]:
```suggestion
 *
```

Comment (Rich-Harris) [reply packages/create-svelte/index.js]: lol whoops

File: packages/create-svelte/templates/default/src/routes/todos/index.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import { api } from './_api';
-import type { RequestHandler } from '@sveltejs/kit';
+import type { RequestHandler } from './index';
```
Comment (star2000) [inline packages/create-svelte/templates/default/src/routes/todos/index.ts:2]: Why RequestHandler is imported from ./index instead of @sveltejs/kit

Comment (gtm-nayan) [reply packages/create-svelte/templates/default/src/routes/todos/index.ts:2]: For those sweet sweet [generated types](https://kit.svelte.dev/docs/types#generated-types)

Comment (star2000) [reply packages/create-svelte/templates/default/src/routes/todos/index.ts:2]: image
image
this confuses me

Comment (dummdidumm) [reply packages/create-svelte/templates/default/src/routes/todos/index.ts:2]: See #4705

Comment (star2000) [reply packages/create-svelte/templates/default/src/routes/todos/index.ts:2]: image
Correct parsing, maybe it's time to merge 4705

**Pull request merged**